lib/cdo/hamburger.rb

```diff
@@ -114,8 +114,15 @@ def self.get_hamburger_contents(options)
       entry[:title] = I18n.t("#{loc_prefix}#{entry[:title]}")
     end.freeze
 
+    # When viewing courses, a signed-out user in English gets the teacher view.
+    if !["teacher", "student"].include?(options[:user_type]) && options[:language] == "en"
```

Comment:
Two things.

1. I imagine the linter wants you to express this array of words as `%w(teacher student)`.

2. Is "not a teacher or a student" really the best way to check for a signed-out user?  We must have a nicer way to express this.